.github/workflows/ruby.yml

```diff
@@ -12,16 +12,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [ '2.6', '2.7', '3.0' ]
+        ruby: [ '2.7', '3.0', '3.1', '3.2' ]
```

Comment:
2.7 is dead for 7 months already, maybe we can drop it too?

Reply:
hm, okay, thanks :)
done